docs/installation/installation-script.md

```diff
@@ -88,11 +88,14 @@ Invoke-Expression (Invoke-WebRequest -Uri "https://get.gpustack.ai" -UseBasicPar
+| `INSTALL_INDEX_URL`               | (empty)                              | Base URL of the Python Package Index.                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `INSTALL_PACKAGE_SPEC`            | `gpustack[all]` or `gpustack[audio]` | The package spec to install. The install script will automatically decide based on the platform. It supports PYPI package names, URLs, and local paths. See the [pip install documentation](https://pip.pypa.io/en/stable/cli/pip_install/#pip-install) for details. <ul><li>`gpustack[all]`: With all inference backends: llama-box, vllm, vox-box.</li><li>`gpustack[vllm]`: With inference backends: llama-box, vllm.</li><li>`gpustack[audio]`: With inference backends: llama-box, vox-box.</li></ul> |
+| `INSTALL_SKIP_POST_CHECK`         | (empty)                              | If set to 1, the installation script will skip the post-installation check.                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `INSTALL_SKIP_BUILD_DEPENDENCIES` | `1`                                  | If set to 1 will skip the build dependencies.                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `INSTALL_SKIP_WIRED_LIMIT`        | (empty)                              | If set to 1 will skip the wired memory reset in macos.                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
INSTALL_SKIP_IOGPU_WIRED_LIMIT
xxx skip setting the GPU wired memory limit on macOS.